pkg/scanner/vulnerability/aqua/converter_test.go

```diff
@@ -0,0 +1,109 @@
+package aqua
+
+import (
+	"os"
+	"testing"
+
+	"github.com/aquasecurity/starboard-security-operator/pkg/etc"
+	"github.com/aquasecurity/starboard/pkg/apis/aquasecurity/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConverter_Convert(t *testing.T) {
+	testCases := []struct {
+		name string
+
+		expectedReport v1alpha1.VulnerabilityReport
+	}{
```

Comment:
Is it possible to get a report with some resources, but no vulnerabilities for those resources?

Reply:
I believe it's not possible because we do not pass the `--full-output` flag to the `scannercli scan` command. However, we do ignore Negligible vulnerabilities, because we do not have such qualitative severity in the vulnerabilities CRD.